__tests__/testinwork.js

```diff
@@ -0,0 +1,18 @@
+// import toJson from 'enzyme-to-json';
+// import React from 'react';
+// import { shallow } from 'enzyme';
+// import HomePage from '../src/components/HomePage';
+
+describe('<HomePage /> shallow rendering', () => {
+  // it('h1 contains correct text', () => {
```

Comment:
No commented out code should land in master.
Either remove it or un-comment it.